pages/12.Plugins/22 GrapesJs/creating a template

```diff
@@ -0,0 +1,79 @@
+# Creating themes for Mautic GrapesJs plugin
```

Comment:
I think this file is not needed if the content is in the Builders section - would suggest removing pages/12.Plugins/22 GrapesJs/creating a template to avoid duplication/confusion?